pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.3.1
 homepage: https://github.com/fluttercandies/extended_image_library
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.8.0 <3.0.0"
```

Comment:
why change this?

Reply:
For analysis backward capability with null-safed packages (https://stackoverflow.com/a/63229503/6858892)

extended_image requires flutter 1.17 - https://github.com/fluttercandies/extended_image/blob/master/pubspec.yaml#L8

flutter 1.17 uses dart 2.8. flutter doctor output example - https://github.com/flutter/flutter/issues/56769